SwiftCompilerSources/Sources/Optimizer/ModulePasses/MandatoryPerformanceOptimizations.swift

```diff
@@ -37,6 +37,7 @@ let mandatoryPerformanceOptimizations = ModulePass(name: "mandatory-performance-
   } else {
     worklist.addAllPerformanceAnnotatedFunctions(of: moduleContext)
     worklist.addAllAnnotatedGlobalInitOnceFunctions(of: moduleContext)
+    worklist.addAllConstGlobalInitOnceFunctions(of: moduleContext)
```

Comment:
Since we have multiple of those `addAll...` functions, I think it makes sense to refactor this and do a single loop over all functions, and add all function which match any of those criteria.